glusterd2/commands/volumes/volume-delete.go

```diff
@@ -37,39 +39,46 @@ func registerVolDeleteStepFuncs() {
 }
 
 func volumeDeleteHandler(w http.ResponseWriter, r *http.Request) {
-
 	ctx := r.Context()
-	logger := gdctx.GetReqLogger(ctx)
```

Comment:
dont we require this logger?

Reply:
Not in volumeDeleteHandler, but is now defined in DeleteVolume.